docs/zh/src/dev-guide/build/build-docker-image.md

```diff
@@ -0,0 +1,195 @@
+# 构建 server docker 镜像
+
+## 构建多架构 docker 镜像
+```bash
+sh ./scripts/docker/buildx.sh
```

Comment:
buildx.sh  does not exist

Reply:
https://github.com/traas-stack/holoinsight/blob/main/scripts/docker/buildx.sh